recipe/meta.yaml

```
@@ -1,83 +1,46 @@
 {% set name = "cuda-nvcc-split" %}
 {% set version = "12.0.76" %}
 {% set cuda_version = "12.0" %}
-{% set platform = "linux-x86_64" %}  # [linux64]
-{% set platform = "linux-ppc64le" %}  # [ppc64le]
-{% set platform = "linux-sbsa" %}  # [aarch64]
-{% set platform = "windows-x86_64" %}  # [win]
-{% set target_name = "x86_64-linux" %}  # [linux64]
-{% set target_name = "ppc64le-linux" %}  # [ppc64le]
-{% set target_name = "sbsa-linux" %}  # [aarch64]
-{% set target_name = "x64" %}  # [win]
-{% set extension = "tar.xz" %}  # [not win]
-{% set extension = "zip" %}  # [win]
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvcc/{{ platform }}/cuda_nvcc-{{ platform }}-{{ version }}-archive.{{ extension }}
-  sha256: 160ca9f8828f53daa4b2066c5361aacb45fa2575885f70c223cda1d11df53d6f  # [linux64]
-  sha256: dc5bd7eac57b32a9645c0c23c6a1ba8a9533e846eeec31414e1aa4230afee2f8  # [ppc64le]
-  sha256: 85c8c643ea2cfa398e1b5ffcd597f5bb3b738526c9e3fb8e39dd909c55345f36  # [aarch64]
-  sha256: f67406f44263f0e4a110a8cf0200bff1d3d0e32917db8a7c9e417a87eb7ca372  # [win]
-  patches:
-    - nvcc.profile.patch      # [linux]
-    - nvcc.profile.patch.win  # [win]
+  # Using this license for consistent messaging with the components it relies on
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvcc/LICENSE.txt
+  sha256: 5db25d4fd138013b563f9a3d1d87f7de7df1dac014fd4cccdfbb3435a5cff761
 
 build:
-  number: 2
+  number: 3
   skip: true  # [osx or win]
+  skip: true  # [build_platform != "linux-64" and build_platform != target_platform]
```

Oops sorry think we need to change this to...

```suggestion
  skip: true  # [target_platform != "linux-64" and target_platform != cross_target_platform]
```